### PR TITLE
Feature/council new test

### PR DESCRIPTION
**Pull request by JacOng17**
# What has changed and why?
Rearranged & added scenarios for creating a new council.

# How to test this change
- [ ] at least one a reviewer ran the code

### CODE REVIEW

File: spec/features/admin/councils/new_spec.rb [hunk 1]
```diff
@@ -66,22 +66,23 @@
...
-      #   expect(page).to have_text('Name has already been taken')
-      #   expect(page).to have_text('Short name has already been taken')
-      # end
+      # Duplicate Name & Short name
+      it do
+        FactoryBot.create :council, name: 'New Council 1', short_name: 'New C1'
+        fill_in 'Name', with: 'New Council 1'
+        fill_in 'Short name', with: 'New C1'
+        fill_in 'Email', with: 'C1@council.co.nz'
+        expect { click_button 'Save' }.to change { Council.count }.by(0)
```
Comment (Br3nda) [inline spec/features/admin/councils/new_spec.rb]:
```suggestion
        expect { click_button 'Save' }.not_to change { Council.count }
```

File: spec/features/admin/councils/new_spec.rb [hunk 2]
```diff
@@ -66,22 +66,23 @@
       # Only Email filled in
       it do
         fill_in 'Email', with: 'C1@council.co.nz'
-        click_button 'Save'
+        expect { click_button 'Save' }.to change { Council.count }.by(0)
```
Comment (Br3nda) [inline spec/features/admin/councils/new_spec.rb]:
```suggestion
         expect { click_button 'Save' }.not_to change { Council.count }
```

File: spec/features/admin/councils/new_spec.rb [hunk 3]
```diff
@@ -33,22 +33,22 @@
       it { expect(page).to have_text('where to send') }
       it { expect(page).to have_unchecked_field('council_active') }
       it { expect(page).to have_text('Active') }
-      it { expect(page).to have_selector(:link_or_button, 'Save') }
-      it { expect(page).to have_selector(:link_or_button, 'Back') }
+      it { expect(page).to have_button('Save') }
+      it { expect(page).to have_link('Back') }
```
Comment (Br3nda) [inline spec/features/admin/councils/new_spec.rb]: :sparkler: :white_flower: :trolleybus: :trophy:

File: spec/features/admin/councils/new_spec.rb (hunk 1 above)
Comment (JacOng17) [reply spec/features/admin/councils/new_spec.rb]: Rubocop throws an error when that is implemented :frowning_face:

File: spec/features/admin/councils/new_spec.rb (hunk 2 above)
Comment (JacOng17) [reply spec/features/admin/councils/new_spec.rb]: Rubocop throws an error when that is implemented :frowning_face:

File: spec/features/admin/councils/new_spec.rb (hunk 3 above)
Comment (JacOng17) [reply spec/features/admin/councils/new_spec.rb]: :smile:

**Pull request merged**